tools/CrossReference.csv

```diff
@@ -1,399 +1,399 @@
+"Adventures","Miser","Miser","Geizhals",,"Miséreux",
+"Adventures","Mission","Mission","Mission",,"Mission",
+"Adventures","Coin of the Realm","Coin of the Realm","Königliche Münzen",,"Monnaie Royale",
+"Adventures","Page","Page","Page",,"Page",
+"Adventures","Peasant","Peasant","Kleinbauer",,"Paysan",
+"Adventures","Pilgrimage","Pilgrimage","Pilgerfahrt",,"Pèlerinage",
+"Adventures","Magpie","Magpie","Elster",,"Pie voleuse",
+"Adventures","Treasure Trove","Treasure Trove","Schatz",,"Pierres précieuses",
+"Adventures","Scouting Party","Scouting Party","Spähtrupp",,"Pistage",
+"Adventures","Plan","Plan","Planung",,"Plan de Bataille",
+"Adventures","Bridge Troll","Bridge Troll","Brückentroll",,"Pont aux Trolls",
+"Adventures","Quest","Quest","Quest",,"Quête",
+"Adventures","Raid","Raid","Überfall",,"Raid",
+"Adventures","Pathfinding","Pathfinding","Wegsuche",,"Reconnaissance",
+"Adventures","Hireling","Hireling","Gefolgsmann",,"Recrue",
+"Adventures","Relic","Relic","Relikt",,"Relique",
+"Adventures","Save","Save","Zuflucht",,"Resserre",
+"Adventures","Seaway","Seaway","Seeweg",,"Route Maritime",
+"Adventures","Soldier","Soldier","Soldat",,"Soldat",
+"Adventures","Swamp Hag","Swamp Hag","Sumpfhexe",,"Sorcière des marais",
+"Adventures","Distant Lands","Distant Lands","Ferne Lande",,"Terres lointaines",
+"Adventures","Transmogrify","Transmogrify","Transformation",,"Transfiguration",
+"Adventures","Port","Port","Hafenstadt",,"Ville portuaire",
+"Alchemy","Alchemist","Alchemist","Alchemist","Alchimista","Alchimiste",
+"Alchemy","Apothecary","Apothecary","Apotheker","Farmacista","Apothicaire",
+"Alchemy","Apprentice","Apprentice","Lehrling","Apprendista","Apprenti",
+"Alchemy","Scrying Pool","Scrying Pool","Vision","Divinazione","Bassin divinatoire",
+"Alchemy","Familiar","Familiar","Vertrauter","Famiglio","Familier",
+"Alchemy","Golem","Golem","Golem","Golem","Golem",
+"Alchemy","Herbalist","Herbalist","Kräuterkundiger","Erborista","Herboriste",
+"Alchemy","Philosopher's Stone","Philosopher's Stone","Stein der Weisen","Pietra Filosofale","Pierre philosophale",
+"Alchemy","Possession","Possession","Besessenheit","Possessione","Possession",
+"Alchemy","Potion","Potion","Trank","Pozione","Potion",
+"Alchemy","Transmute","Transmute","Verwandlung","Trasmutazione","Transmutation",
+"Alchemy","University","University","Universität","Università","Université",
+"Alchemy","Vineyard","Vineyard","Weinberg","Vigna","Vignoble",
+"Base","Silver","Silver","Silber","Argento","Argent","Měďák"
+"Base","Copper","Copper","Kupfer","Rame","Cuivre","Stříbrňák"
+"Base","Estate","Estate","Anwesen","Tenuta","Domaine","Statek"
+"Base","Duchy","Duchy","Herzogtum","Ducato","Duché","Vévodství"
+"Base","Curse","Curse","Fluch","Maledizione","Malédiction","Kletba"
+"Base","Gold","Gold","Gold","Oro","Or","Zlaťák"
+"Base","Province","Province","Provinz","Provincia","Province","Provincie"
+"Base","Trash","Trash","Müll","Carte Eliminate","Smetiště"

```

Comment:
Starting here, a bunch of lines only have 6 fields instead of 7. All lines should have 7 now.